src/ParticleContainer.h

```diff
@@ -0,0 +1,48 @@
+/**
+ * @file
+ *
+ * @brief The ParticleContainer is in charge of storing the particles as well as there molecular bonds.
```

Comment:
```suggestion
 * @brief The ParticleContainer is in charge of storing the particles as well as their molecular bonds.
```
Fyi, I don't care about spelling/grammar in terms of grading (within reason) - it just bothered me personally :stuck_out_tongue_winking_eye: